### PR TITLE
Rename `UnIt_SampleApp` to `UnItSpecsHost` #19

### PR DESCRIPTION
**Pull request by connected-akhan**

### CODE REVIEW

**Comment by connected-swu** [comment]
Hmmm, all deletions and no additions?

**Comment by connected-akhan** [comment]
> Hmmm, all deletions and no additions?

Good point... I thought all changes were staged before commit, because `git add -p` returned `No changes` but, after double checking, and via `git status`, I see a the `UnItSpecsHost` file as untracked for some reason. Must've happened at some point during all the renaming.

**Comment by connected-akhan** [comment]
@connected-swu Added the project file! That looks more like it.

File: UnItSpecsHost/Carthage/Checkouts/Nimble/.github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,36 @@
+- [ ] I have read [CONTRIBUTING](https://github.com/Quick/Nimble/blob/master/CONTRIBUTING.md) and have done my best to follow them.
```
Comment (connected-swu) [inline UnItSpecsHost/Carthage/Checkouts/Nimble/.github/ISSUE_TEMPLATE]: I don't think we want to include anything in the `Carthage` folder – these are temporary build artefacts that shouldn't be committed – it feels strange that the folder is sitting under `UnItSpecsHost`.

File: UnItSpecsHost/UnItSpecsHost.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -665,7 +665,7 @@
 			defaultConfigurationIsVisible = 0;
 			defaultConfigurationName = Release;
 		};
-		F424F1C921E6472E00F2888C /* Build configuration list for PBXNativeTarget "UnIt_SampleAppTests" */ = {
+		F424F1C921E6472E00F2888C /* Build configuration list for PBXNativeTarget "UnItSpecsHostTests" */ = {
 			isa = XCConfigurationList;
 			buildConfigurations = (
```
Comment (connected-swu) [inline UnItSpecsHost/UnItSpecsHost.xcodeproj/project.pbxproj]: Note to self, this is where the xcode project file changes are.

File: UnItSpecsHost/Carthage/Checkouts/Nimble/.github/ISSUE_TEMPLATE (hunk 1 above)
Comment (connected-akhan) [reply UnItSpecsHost/Carthage/Checkouts/Nimble/.github/ISSUE_TEMPLATE]: >  it feels strange that the folder is sitting under UnItSpecsHost 

When the app was called UnIt_SampleApp, the Carthage folder was under the UnIt_SampleApp folder as well. So the location of the folder did not change in this commit. 

Screen Shot 2019-08-06 at 10 04 51 AM

**Comment by connected-akhan** [comment]
> ❌ don't commit `Carthage` folder items

I removed it from the commit by `git rm -r --cached Carthage` and then committing again.

**Comment by connected-swu** [comment]
Checked out branch, and quickly ran,
```sh
grep -rnw '.' -e 'UnIt_SampleApp'
```

and got,

```
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:18:               BuildableName = "UnIt_SampleApp.app"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:19:               BlueprintName = "UnIt_SampleApp"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:20:               ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:38:               ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:48:               ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:56:            BuildableName = "UnIt_SampleApp.app"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:57:            BlueprintName = "UnIt_SampleApp"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:58:            ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:79:            BuildableName = "UnIt_SampleApp.app"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:80:            BlueprintName = "UnIt_SampleApp"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:81:            ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:98:            BuildableName = "UnIt_SampleApp.app"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:99:            BlueprintName = "UnIt_SampleApp"
./UnIt_SampleApp/UnIt_SampleApp.xcodeproj/xcuserdata/cl-dev.xcuserdatad/xcschemes/UnIt_SampleApp.xcscheme:100:            ReferencedContainer = "container:UnIt_SampleApp.xcodeproj">
./UnItSpecsHost/UnItSpecsHost/Base.lproj/Main.storyboard:16:                <viewController storyboardIdentifier="ViewController" id="BYZ-38-t0r" customClass="ViewController" customModule="UnIt_SampleApp" customModuleProvider="target" sceneMemberID="viewController">
./UnItSpecsHost/UnItSpecsHost/Base.lproj/Main.storyboard:120:                                            <collectionViewCell opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" reuseIdentifier="firstCollectionView" id="FdK-RB-89z" customClass="FirstCollectionViewCell" customModule="UnIt_SampleApp" customModuleProvider="target">
./UnItSpecsHost/UnItSpecsHost/Base.lproj/Main.storyboard:137:                                                                <collectionViewCell opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" restorationIdentifier="SecondCollectionViewCell" reuseIdentifier="secondCollectionViewCell" id="ljG-FG-Acu" customClass="SecondCollectionViewCell" customModule="UnIt_SampleApp" customModuleProvider="target">
./UnItSpecsHost/UnItSpecsHost/ViewControllers/OverlapViewController.xib:13:        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="OverlapViewController" customModule="UnIt_SampleApp" customModuleProvider="target">
./UnItSpecsHost/UnItSpecsHost.xcodeproj/project.xcworkspace/contents.xcworkspacedata:5:      location = "self:/Users/amaan/Workspace/unit5/UnIt/UnIt_SampleApp/UnItSpecsHost.xcodeproj">
Binary file ./UnIt.xcworkspace/xcuserdata/cl-dev.xcuserdatad/UserInterfaceState.xcuserstate matches
```

☝️ not sure if that's cause for concern?

Also, merge conflicts =P

**Comment by connected-akhan** [comment]
Closing this PR. Used grep method in PR #38

**Pull request closed without merging**